### PR TITLE
Adding poster to Brightcove videos

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
e.g.

screen shot 2015-03-04 at 14 23 06

fixes https://github.com/Financial-Times/next-grumman/issues/175

### CODE REVIEW

File: client/components/video/main.js [hunk 1]
```diff
@@ -24,9 +24,13 @@ function brightcoveInit(el) {
...
 	var videoId = url.slice((url.lastIndexOf('/')+1));
 
-	brightcove(videoId).then(function(url){
+	brightcove(videoId).then(function(data) {
 		var videoEl = document.createElement('video');
-		videoEl.setAttribute('src', url);
+		videoEl.setAttribute('src', data.src);
+		videoEl.setAttribute(
+			'poster',
+			'//image.webservices.ft.com/v1/images/raw/' + data.poster + '?width=690&source=grumman&fit=scale-down'
```
Comment (ironsidevsquincy) [inline client/components/video/main.js]: might be nice to split this into a helper, shared with handlebar's `resize`

Comment (matthew-andrews) [inline client/components/video/main.js]: :+1:

**Comment by commuterjoy** [comment]
+1 would be nice for someone to look in to the quality of the poster images. fixed text on an image is a bit lame.

File: client/components/video/main.js (hunk 1 above)
Comment (matthew-andrews) [inline client/components/video/main.js]: you migth need to uriencode the data.poster…  not sure…

**Pull request merged**